composer.json

```diff
@@ -22,7 +22,7 @@
     ],
     "require": {
         "php": "^7.2|^8.0",
-        "honeybadger-io/honeybadger-php": "^2.14.1",
+        "honeybadger-io/honeybadger-php": "dev-checkins-manager",
```

Comment:
Did you mean to check this in?

Reply:
Yeah, that's one thing I dislike about the composer local surrogacy—you have to remember to not check it in. Also the repositories entry below.

Reply:
Before merging, I will set the correct dependency version, but it has to be after we merge and release the honeybadger-php package.